seeds/vocabulary/_sources.yml

```diff
@@ -0,0 +1,34 @@
+seeds:
```

Comment:
I'd been putting seeds in the project file (https://github.com/OHDSI/dbt-synthea/blob/952fee45e89959487cb1c6d81bbd7e2ba2e38bbb/dbt_project.yml#L24).  I didn't realize they could go elsewhere!  I prefer it together with the seeds like this.  But curious why the yml syntax differs here vs what I put in the project file?

Reply:
Maybe just multiple ways to write the same thing.  Either way, down to move over all the vocab stuff into this file here if that works.

Reply:
On second thought, maybe this config should stay in the project file:

```
seeds:
  synthea_omop_etl:
    map:
      +enabled: true
      +schema: map_seeds
    vocabulary:
      +enabled: true
      +schema: vocab_seeds
    synthea:
      +enabled: true
      +schema: synthea_seeds
```

And then the column types could live alongside the seeds.  (In which case no change needed here.)

LMK what you think.

Reply:
> I didn't realize they could go elsewhere! I prefer it together with the seeds like this. But curious why the yml syntax differs here vs what I put in the project file?

No idea 😆 It caught me out too... The docs [just point out they're different](https://docs.getdbt.com/reference/seed-configs) without saying why.

> LMK what you think

Very good question - I think the split approach is best in this case; although I would be happy with either solution. I think keeping the more global seeds config in the `projects` file, and then the implementation of how the seeds are ingested is a good split of responsibility (keep specific config near the code/sources). But agree this may be confusing to people coming to the project.

I think this is a best balance but not super married to it if you'd rather unify where it to one place

Reply:
OK yeah I agree with the split approach!  I always thought it was weird listing all those types out in the project file.